ci/install_libseccomp.sh

```diff
@@ -70,7 +70,8 @@ build_and_install_gperf() {
     curl -sLO "${gperf_tarball_url}"
     tar -xf "${gperf_tarball}"
     pushd "gperf-${gperf_version}"
-    ./configure --prefix="${gperf_install_dir}"
+    # Unset $CC for configure, we will always use native for gperf
```

Comment:
Sorry, didn't get. Why?

Reply:
> the use of an environment called `CC`. Today we export `export CCV0="yes"` to mean "enable CC" stuffs; at some point in time `CCV0` becomes `CC` and, bang, we have a naming clash. :(

`CC` (C compiler) is an environment variable used by autotools, so we should avoid it for Confidential Containers. This is also what it is used for here, and this matters e.g. for libseccomp, where we want to build it for the target architecture, and `CC` would be something like `s390x-linux-gnu-gcc`.

> Sorry, didn't get. Why?

Here specifically, we build gperf against the native architecture, as it is a build dependency for libseccomp, and not for the target. Thus, we unset `CC`, which might point to a cross compiler. I can clarify the comment a little.

Reply:
Could you summarise that in a comment in the file itself as the existing comment is a bit too terse to derive all that ;)

Also, if this is CC specific, we should state that (which reminds me of the rather unfortunate issue of "CC" being an overloaded acronym in this instance).

Reply:
This isn't CC specific -- it's a port of #3682. I'll port all additional comments gathered from the review here back into main.